doc/source/docker/run-quickstart-examples-docker-compose.rst

```diff
@@ -0,0 +1,114 @@
+Run Flower Quickstart Examples with Docker Compose
+==================================================
+
+Flower provides a set of `quickstart examples <https://github.com/adap/flower/tree/main/examples>`_
+to help you get started with the framework.
+
+These examples are designed to demonstrate the capabilities of Flower and can be run
+using Docker Compose.
+
+.. important::
+
+   Some quickstart examples may have limitations or requirements that prevent them from running
+   on every environment. For more information, please see `Limitations`_.
+
+Prerequisites
+-------------
+
+Before you start, make sure that:
+
+- The ``flwr`` CLI is :doc:`installed <../how-to-install-flower>` locally.
+- The Docker daemon is running.
+- Docker Compose is `installed <https://docs.docker.com/compose/install/>`_.
+
+Run the Quickstart Example
+--------------------------
+
+#. Clone the quickstart example you like to run. For example, ``quickstart-sklearn-tabular``:
+
+   .. code-block:: bash
+
+      $ git clone --depth=1 https://github.com/adap/flower.git \
+           && mv flower/examples/quickstart-sklearn-tabular . \
+           && rm -rf flower && cd quickstart-sklearn-tabular
+
+#. Download the `compose.yml <https://github.com/adap/flower/blob/main/src/docker/complete/compose.yml>`_ file into the example directory:
+
+   .. code-block:: bash
+
+      $ curl https://raw.githubusercontent.com/adap/flower/refs/heads/main/src/docker/complete/compose.yml \
+          -o compose.yml
+
+#. Build and start the services using the following command:
+
+   .. code-block:: bash
+
+      $ docker compose up --build -d
+
+#. Append the following lines to the end of the ``pyproject.toml`` file and save it:
+
+   .. code-block:: toml
+      :caption: pyproject.toml
+
+      [tool.flwr.federations.docker-compose]
```

Comment:
I'm thinking if we should name the federation to something different that doesn't use the `docker` or `	compose` keywords. The alternative would be to add a comment saying that the string that comes after `federations.` is up to the user to define (then use is accordingly when doing `flwr run .`

Reply:
I'm open to alternative name suggestions. we can also call it `local` or `local-insecure` 

I'll add a comment, but this i feel this information belongs more to the flwr cli docs.

Reply:
how about `local-deployment`? I agree info should better be left to other parts of the docs.

Reply:
`local-deployment` sounds good to me!